packages/fuels-core/src/code_gen/functions_gen.rs

```diff
@@ -149,13 +149,9 @@ fn expand_fn_outputs(outputs: &[Property]) -> Result<TokenStream, Error> {
             }
         }
         // Recursively expand the outputs
```

Comment:
Maybe remove this comment now that we can finally not care about multiple return values 😂 
```suggestion
```